site/docs/00-z-quick-start.mdx

```diff
@@ -8,7 +8,7 @@ Excalibur uses a theater-style metaphor to organize your games. There are `Actor
 The fastest way to get an Excalibur project up and running is using the npm CLI tool. All of the included [templates are open source on our github](https://github.com/excaliburjs/?q=template-&type=all&language=&sort=#org-profile-repositories)
 
 ```sh
-npx create-excalibur@latest
+npm init excalibur 
```

Comment:
need to keep `@latest` in the command otherwise npm may use an older cached version if you've run it before